scripts/prepare_sector_network.py

```diff
@@ -1066,7 +1083,7 @@ def add_industry(n, costs):
         carrier="gas for industry",
     )
 
-    gas_demand = industrial_demand.loc[nodes, "methane"] / 8760.0
+    gas_demand = industrial_demand.loc[nodes, "gas"] / 8760.0

```

Comment:
Is there any specific reason you changed "methane" -> "gas"? "Gas" is ambiguous to me

Reply:
Also the cost and efficiency parameters we currently take from PyPSA-EUR data name it as "gas". So just for the sake of consistency.
In general, I like the wording "methane" better like you but I think we are already using gas in many different places.

Reply:
Alright, that makes sence. Especially the consistency with PyPSA-Eur.